docs/user/SUMMARY.md

```diff
@@ -3,6 +3,7 @@
 # User Guides
 
 - [Adding Glean to your project](user/adding-glean-to-your-project.md)
+  - [C#](user/adding-glean-to-your-project/csharp.md)
```

Comment:
FWIW, we have actually removed C# documentation as we did the restructuring on other pages.

Reply:
Oh. Ok, let me remove the docs here too, but I'm not sure we should have done that if the code is still around

Reply:
Yeah, to be honest I am not sure either after thinking about it a bit more. It is weird though to simply leave the documentation normally. The bug the C# page links too is closed as WONTFOX as are all the other C# bugs.